source/deployment/guest-accounts.rst

```diff
@@ -79,65 +122,95 @@ Managing Guests
 
-System Administrators can also promote a *Guest* to *Member* by updating their role in **System Console > User Management > Users**.  
+System Administrators can demote a *Member* to *Guest* by updating their role in **System Console > User Management > Users**. The demoted user will retain their existing channel and team memberships but will be restricted from discovering public channels and collaborating with users outside of the channels they are in.  This is useful if you are already collaborating with external contractors and want to restrict their abilities within your Mattermost instance.
+
+System Administrators can also promote a *Guest* to *Member* by updating their role in **System Console > User Management > Users**.
 
-**Note:** You can filter the list in **System Console >  User Management > Users** to view all guests on the system. 
+**Note:** You can filter the list in **System Console >  User Management > Users** to view all guests on the system.
 
 Disabling Guest Accounts
 ------------------------
 
-To disable the Guest Accounts feature, go to **System Console > Authentication > Guest Access > Enable Guest Access** and select ``False``. Current Guest Accounts will remain active until guest users are manually marked ``inactive`` in **System Console > User Management > Users**. 
+To disable the Guest Accounts feature, go to **System Console > Authentication > Guest Access > Enable Guest Access** and select ``False``. In versions
+prior to 5.18, current Guest Accounts will remain active until guest users are manually marked ``inactive`` in **System Console > User Management > Users**.
+
+From 5.18, if you are using AD/LDAP and the Guest Access setting is disabled, the Guest Filter and existing guest users in System Console are deactivated.
+Additionally, no new guests can be invited or added using the filter as an authentication method. If a previous guest's
+credentials match the user filter (the only filter which is active when Guest Access is disabled), they will be reactivated and promoted to a member user
+upon their next login.
+
+Similarly, for SAML, when the Guest Access setting is disabled, the Guest Attribute and existing guest users in System Console are deactivated.
+Additionally, no new guests can be invited or added using the attribute as an authentication method. If a previous guest's
+credentials match the user attribute (the only attribute which is active when Guest Access is disabled), they will be reactivated and promoted
+to a member user upon their next login.
 
-You can disable individual guest accounts in **System Console > User Management**, via the **Manage Members** modal, or using the ``/kick`` or ``/remove`` commands. 
+You can disable individual guest accounts in **System Console > User Management**, via the **Manage Members** modal, or using the ``/kick`` or ``/remove`` commands. In version 5.18,
```

Comment:
Does the ``kick`` and ``remove`` commands actually disable the account or remove them from teams/channesl?

Reply:
That's a good question. I don't actually know - it's in the current docs, I may just have moved this around a bit and didn't review it. The names of the commands makes me think the latter though.

Reply:
Confirmed both commands remove a user from a channel.  No deactivation and their membership to the team is retained.

Reply:
Updated the docs to reflect this.